server/device-sso-auth-lsp/README.md

```diff
@@ -0,0 +1,49 @@
+# Device Auth Language Server
+
+This is an example implementation of Device authentication SSO flow for AWS Language Servers project.
+
+It is port of [SSO flow implementation in VSCode sample client](../../client/vscode/src/sso/builderId.ts), but as LSP server.
+
+[`SsoAuthServer`](./src/language-server/SsoAuthServer.ts) Server implementation is the entry point for this Server. 
+
+Supports only [`standalone`](https://github.com/aws/language-server-runtimes/blob/main/runtimes/runtimes/standalone.ts) AWS Server Runtime, as it requires NodeJS `fs` access.
```

Comment:
I wonder if we could cache the token in memory instead of the fs for the webworker implementation?

Reply:
VSCode sample caches in memory. Caching on filesystem solves the need to run Auth flow between client restarts - caching in memory loses token after Language server is closed, which is not useful in real-life usage scenario